ert3/data/_record.py

```diff
@@ -5,12 +5,28 @@
 import uuid
 from abc import abstractmethod
 from enum import Enum, auto
+from functools import partial, wraps
 from pathlib import Path
 from typing import Awaitable, List, Mapping, Tuple, Union
 
+import aiofiles
 from pydantic import BaseModel, root_validator
 
 
+def wrap(func):
+    @wraps(func)
+    async def run(*args, loop=None, executor=None, **kwargs):
+        if loop is None:
+            loop = asyncio.get_event_loop()
+        pfunc = partial(func, *args, **kwargs)
+        return await loop.run_in_executor(executor, pfunc)
+
+    return run
+
+
+_copy = wrap(shutil.copy)
```

Comment:
I see we read the same issue 😄 
```suggestion
import aiofiles.os

_copy = aiofiles.os.wrap(shutil.copy)
```

Reply:
😎
Problem is that they don't export wrap